lib/utils/combineJSON.js

```diff
@@ -48,6 +48,34 @@ function combineJSON(arr, deep, collision) {
       throw e;
     }
 
+    if(useFilePathAsCTI) {
+      let filePath = path.relative('./', files[i]);
```

Comment:
I think it might be too restricting to assume that the names should start from the current working directory. For example we would run the command from our package root, but our properties were in `<root>/src/tokens/<properties>`. In that case everything would be prefixed with "srcTokens". What if "rootPath" or something was a configuration option? That would also solve the issue you commented about on line 59 with assuming a "properties" folder name.

Reply:
👍🏽 Good call on all accounts